.github/workflows/deploy_aws_api_infrastructure_dev.yml

```diff
@@ -17,21 +17,26 @@ defaults:
   run:
     working-directory: api/awsauth/
 
+env:
```

Comment:
This let's us run the action off of branches other than `main` (so we can test the dev environment)